app/ledger.js

```diff
@@ -383,6 +391,9 @@ eventStore.addChangeListener(() => {
 var initialize = (onoff) => {
   enable(onoff)
 
+  // Check if relevant browser notifications should be shown every 15 minutes
+  setInterval(showNotifications, 15 * msecs.minute)
```

Comment:
maybe add a global var to keep track of whether `setInterval` has already been called and do nothing here if it has